tests/drucker_dashboard/test_kubernetes.py

```diff
@@ -68,6 +71,17 @@ def test_post(self):
         ).one_or_none()
         self.assertIsNotNone(kobj)
 
+        # Check Kubernetes dump files
+        aobj = Application.query.filter_by(
+            kubernetes_id=kobj.kubernetes_id).one_or_none()
+        if aobj is not None:
```

Comment:
Can't we assert `aobj` is not `None` here?

Reply:
No we can't.
I have noticed that this `aobj` is always `None`.
So we can remove this `# Check Kubernetes dump files` section.

Reply:
Yeah, let's remove this section!